npm/ipsm/ipsm.go

```diff
@@ -28,14 +28,15 @@ type IpsetManager struct {
 // Ipset represents one ipset entry.
 type Ipset struct {
 	name       string
-	elements   []string
+	elements   map[string]string // key = ip, value: context associated to the ip like podOwnername
```

Comment:
Does this podOwnerName mean PodUid mentioned in design doc?

Reply:
Yes, I will fix the comment.